lms/templates/cme_register.html

```diff
@@ -354,7 +347,7 @@ <h3 class="message-title">Welcome ${extauth_id}</h3>
          
           <li class="field select" id="specialty-container">
             <label for="specialty">Specialty</label>
-            <select id="specialty" name="specialty" class="controller select-controller">
```

Comment:
Just grepped around in the lms directory, and there don't appear to be any references to ".controller" in .css or .scss files.